src/main/java/core/basesyntax/fruittransact/FruitTransactionService.java

```diff
@@ -0,0 +1,38 @@
+package core.basesyntax.fruittransact;
+
+import java.util.List;
+
+public class FruitTransactionService {
+    private static final int INDEX_OF_TYPE = 0;
+    private static final int INDEX_OF_FRUIT = 1;
+    private static final int INDEX_OF_AMOUNT = 2;

```

Comment:
you can avoid OF by simply changing the word order
```suggestion
    private static final int TYPE_INDEX = 0;
    ...
```